scripts/generate-headers

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+PROJECT_DIR="$SCRIPT_DIR/../"
+cd $PROJECT_DIR
+
+DEFAULT_OUT="`realpath $PROJECT_DIR`/target/headers"
```

Comment:
`realpath` needs `brew install coreutils` on Macos, and other solutions like `readlink -f` don't work on macos/BSD -- this is only really needed since it's displayed in the `usage`. do you have any cross-platform preferences for something like this? @justinabrahms